Meter.c

```diff
@@ -13,8 +13,10 @@ in the source distribution for its full text.
 #include <float.h>
 #include <limits.h> // IWYU pragma: keep
 #include <math.h>
+#include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#include <wchar.h>
```

Comment:
Remove this include line. `wchar.h` is better included indirectly via `ProvideCurses.h`.